cmd/agent/app/reporter/flags.go

```diff
@@ -35,15 +40,45 @@ type Type string
 // Options holds generic reporter configuration.
 type Options struct {
 	ReporterType Type
+	AgentTags    map[string]string
 }
 
 // AddFlags adds flags for Options.
 func AddFlags(flags *flag.FlagSet) {
 	flags.String(reporterType, string(GRPC), fmt.Sprintf("Reporter type to use e.g. %s, %s", string(GRPC), string(TCHANNEL)))
+	flags.String(agentTags, "", fmt.Sprintf(`One or more tags to be added to the Process tags of all spans passing through this agent.
+		Ex: --jaeger.tags=key1=value1,key2=${envVar:defaultValue}`))
```

Comment:
I would remove `--jaeger.tags=` prefix, limit example to the sample value string `key1=value1,key2=${envVar:defaultValue}`